requirements.txt

```diff
@@ -10,6 +10,9 @@ tensorflow>=2.2
 tfa-nightly==0.12.0.dev20200820045606
 PyYAML>=3.12
 absl-py
+requests
+pypng
```

Comment:
This should be in `requirements_image.txt`

Reply:
Moved